### ISSUE TITLE
Crash on starting

### ISSUE THREAD

**Issue by tarokooong** [mipen/BannerlordTweaks#102]
The only two mods I use are Tweaks and ModLib. I have arranged the mods in the correct order, and all the DLLs are unblocked. I also deleted the older versions before installing the newer versions every time. How can I fix this problem? I have faced this problem since the separation of Tweaks and ModLib.
![Capture](https://user-images.githubusercontent.com/65030394/81420201-d17e3100-9179-11ea-91eb-8445f2501b49.PNG)
![Capture2](https://user-images.githubusercontent.com/65030394/81420205-d2af5e00-9179-11ea-8b05-b393f710c05e.PNG)

**Comment by mipen** [comment]
What language is your computer using?
The error says there are characters that it doesn't recognise in the path to your documents folder. Can you please post the path to you documents folder?

**Comment by tarokooong** [comment]
You are correct. It seems like it is because the location name is in 'Thai,' but I don't understand why can't I reset it back to default.
![Capture](https://user-images.githubusercontent.com/65030394/81436551-e23ba080-9193-11ea-998e-bc7774b83eb8.JPG)
![Capture2](https://user-images.githubusercontent.com/65030394/81436555-e36ccd80-9193-11ea-9f71-12983945cf3a.JPG)
![Capture3](https://user-images.githubusercontent.com/65030394/81436558-e49dfa80-9193-11ea-9064-e31df84aa641.JPG)
